jlm/rvsdg/reduction-helpers.hpp

```diff
@@ -52,7 +52,7 @@ pairwise_reduce(Container && args, const Reductor & reductor)
 {
```

Comment:
Would suggest removing the && and just use plain value. It is up to caller to move values into it if it wants to pass ownership instead of copying.

Reply:
Fixed